models/export.js

```diff
@@ -0,0 +1,85 @@
+/* global JsonRoutes */
+if(Meteor.isServer) {
+  JsonRoutes.add('get', '/api/b/:boardId/:userId/:loginToken', function (req, res) {
+    const { userId, loginToken, boardId } = req.params;
```

Comment:
Maybe add the link to the https://blog.kayla.com.au/server-side-route-authentication-in-meteor/ article to explain to future readers what is happening here